File: java/ex3_1_1.java
class A {
    static <T> T choose(T v1, T v2) { return true ? v1 : v2; }
}


class Example {
    public static void main(String[] args) {
        String str = A.<String>choose("Korean", "Foreigner");
        int num = A.<Integer>choose(1, 2);
    }
}

File: rust/ex3_1_1.rs
fn choose<T>(v1: T, v2: T) -> T { if true { v1 } else { v2 } }

fn main() {
    let str: String = choose::<String>("Korean".to_string(), "Foreigner".to_string());
    let num: i32 = choose::<i32>(1, 2);
}
